src/main/java/uk/gov/hmcts/reform/translate/data/DictionaryEntity.java

```diff
@@ -15,6 +17,8 @@
 
 @Table(name = "dictionary")
 @Entity
+@Builder
+@AllArgsConstructor
 @NoArgsConstructor
 @Data
```

Comment:
We are using 2 lombok annotations to add generated code for all propouses for instance : @AllArgsConstructor
@NoArgsConstructor they are the opposite . May be @Data is good enough

Reply:
I will do  these changes on my branch.